pkgs/top-level/all-packages.nix

```diff
@@ -25669,8 +25669,6 @@ with pkgs;
 
   xsimd = callPackage ../development/libraries/xsimd { };
 
-  xsimd10 = callPackage ../development/libraries/xsimd/10.nix { };
```

Comment:
Don't forget to add a corresponding throw.

Reply:
> Don't forget to add a corresponding throw.

I was thinking about it, but maybe it is not needed as this package existed in Nixpkgs only for 6 weeks - not part of the last NixOS release.